OM-Demo/BridgingHeader.h

```diff
@@ -0,0 +1,18 @@
+//
+//  BridgingHeader.h
+//  OM-Demo
+//
+//  Created by Justin Hines on 8/15/19.
+//  Copyright © 2019 Open Measurement Working Group. All rights reserved.
+//
+
+// TODO: Remove this file once lib is built as a framework
```

Comment:
Is this file necessary now?

Reply:
No, I can remove. Just using it for testing.

Reply:
cool thanks!